From 15c88211a8b7587becac1c5151ccf3677988eb81 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Wed, 9 Aug 2017 13:20:06 +0800
Subject: [PATCH 2356/5242] MLK-16170 ARM64: dts: imx8mq-evk: disable MIPI DSI
 by default

commit  720dd38457a1f1bc18e525bd6fdb01f645700609 from
https://source.codeaurora.org/external/imx/linux-imx.git

The default display for IMX8MQ platform is 'DCSS-->HDMI',
so disable MIPI DSI by default on IMX8MQ.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 562c7e4..9074813 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -520,7 +520,7 @@
 &mipi_dsi {
 	reset = <&src>;
 	mux-sel = <&gpr>;	/* lcdif or dcss */
-	status = "okay";
+	status = "disabled";
 	port {
 		mipi_dsi_ep: endpoint {
 			remote-endpoint = <&dsi_to_hdmi>;
-- 
1.7.9.5

